gnomemusic/notification.py

```diff
@@ -105,7 +105,10 @@ def _update_track(self, player):
                                                              '<i>' + album + '</i>'),
                                       'gnome-music')
 
-            self._notification.show()
+            try:
+                self._notification.show()
+            except:
```

Comment:
We shouldn't trap all the errors here - please limit those to GLib.GError

A proper bug and a patch in bugzilla would be good too